packages/picasso/src/TopBarMenu/TopBarMenu.tsx

```diff
@@ -0,0 +1,43 @@
+import { BaseProps } from '@toptal/picasso-shared'
+import React, { forwardRef, HTMLAttributes, ReactNode } from 'react'
+import { Theme, makeStyles } from '@material-ui/core/styles'
+
+import Menu from '../Menu'
+import { usePortalToHamburger } from '../PageHamburger'
+import styles from './styles'
+
+export interface Props extends BaseProps, HTMLAttributes<HTMLUListElement> {
+  /** Menu content */
+  children: ReactNode
+}
+
+const useStyles = makeStyles<Theme>(styles, { name: 'PicassoTopBarMenu' })
+
+export const TopBarMenu = forwardRef<HTMLUListElement, Props>(
+  function TopBarMenu(props, ref) {
+    const { children, ...rest } = props
+    const classes = useStyles()
+
+    // not more than 6 menu items are allowed by BASE design
+    const items = React.Children.map(children, (child, index) =>
+      index < 6 ? child : null
+    )
```

Comment:
NIT: Maybe use `React.Children.toArray`, wdyt? One of the specific use cases for it is slicing child elements
```suggestion
    const items = React.Children.toArray(children).slice(0, 6)
```

Reply:
Looks much better 🙇 thanks